jdi-light-mobile-examples/src/main/java/selenium/seleniumPO/SiteSelenium.java

```diff
@@ -0,0 +1,19 @@
+package selenium.seleniumPO;
+
+import selenium.entities.User;
+
+import static com.epam.jdi.light.driver.WebDriverFactory.getDriver;
+import static com.epam.jdi.light.elements.init.PageFactory.initElements;
+
+public class SiteSelenium {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'SiteSelenium' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5104364)